api/urls.py

```diff
@@ -76,6 +77,11 @@
         AccountPayoutsReceivedAPI.as_view(),
         name="accounts_api_by_id_payouts_received",
     ),
+    path(
+        "v1/accounts/<str:account_id>/registrations",
```

Comment:
let's change to `"v1/accounts/<str:account_id>/list-registrations"` just to be extra clear, as we may have other kinds of registrations in the future